src/lib/stable/ic_apis/notify.ts

```diff
@@ -0,0 +1,42 @@
+import { IDL, Principal } from '../';
+
+/**
+ * Performs a cross-canister call without awaiting the result
+ * @param canisterId
+ * @param method
+ * @param argsRaw
+ * @param payment
+ * @returns
+ */
+export function notify(
+    canisterId: Principal | string,
+    method: string,
+    options?: {
+        paramIdls?: IDL.Type[];
+        args?: any[];
+        payment?: bigint;
+    }
+): void {
+    if (globalThis._azleIc === undefined) {
```

Comment:
Is there any way we could just define it as a javascript function in rust in the first place, the same way as we make the _azleIc ojbect, but just as functions?

Reply:
We have created an issue to to consider these kinds of broader design concerns before 1.0: https://github.com/demergent-labs/azle/issues/1862